t/01-pdk/04-request/19-get_forwarded_prefix.t

```diff
@@ -0,0 +1,77 @@
+use strict;
+use warnings FATAL => 'all';
+use Test::Nginx::Socket::Lua;
+use t::Util;
+
+$ENV{TEST_NGINX_CERT_DIR} ||= File::Spec->catdir(server_root(), '..', 'certs');
+$ENV{TEST_NGINX_NXSOCK}   ||= html_dir();
+
+plan tests => repeat_each() * (blocks() * 3);
+
+run_tests();
+
+__DATA__
```

Comment:
Should we include a case where the IP is not trusted as well?

Reply:
Added one test case covering that. Since the not-trusted case uses a value set by the router / runloop handler, the value here is simply nil.